docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
+
+The OpenStack CLI tool, also known as OpenStack Command Line Client, 
+conveniently provides access to various OpenStack APIs. Using the 
+OpenStack CLI tool, you can remotely create and manage the lifecycle of 
+objects relating, for example, to Compute, Networking, and Storage.
```

Comment:
"relating" sounds out of place here; what did you mean?

Reply:
Ooops, I mean "related to" :/